<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>974</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>473</width>
         <height>442</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>974</width>
     <height>22</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionNuevo"/>
    <addaction name="actionAbrir"/>
    <addaction name="actionRecientes"/>
    <addaction name="actionCerrar"/>
    <addaction name="separator"/>
    <addaction name="actionExportar"/>
    <addaction name="actionImprimir"/>
    <addaction name="separator"/>
    <addaction name="actionGuardar"/>
    <addaction name="actionGuardar_Como"/>
    <addaction name="separator"/>
    <addaction name="actionSobre_Nosotros"/>
    <addaction name="actionConfiguracion"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuEditar">
    <property name="title">
     <string>Editar</string>
    </property>
    <addaction name="actionCopiar"/>
    <addaction name="actionPegar"/>
    <addaction name="actionCortar"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionTutorial"/>
    <addaction name="actionSobre_Nosotros_2"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuEditar"/>
   <addaction name="menuAyuda"/>
  </widget>
  <action name="actionNuevo">
   <property name="icon">
    <iconset resource="../resources/img.qrc">
     <normaloff>:/images/icons/new.svg</normaloff>:/images/icons/new.svg</iconset>
   </property>
   <property name="text">
    <string>Nuevo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="icon">
    <iconset resource="../resources/img.qrc">
     <normaloff>:/images/icons/save.svg</normaloff>:/images/icons/save.svg</iconset>
   </property>
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionGuardar_Como">
   <property name="icon">
    <iconset resource="../resources/img.qrc">
     <normaloff>:/images/icons/save_as.svg</normaloff>:/images/icons/save_as.svg</iconset>
   </property>
   <property name="text">
    <string>Guardar Como</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionSobre_Nosotros">
   <property name="icon">
    <iconset resource="../resources/img.qrc">
     <normaloff>:/images/icons/info.svg</normaloff>:/images/icons/info.svg</iconset>
   </property>
   <property name="text">
    <string>Sobre Nosotros</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::AboutRole</enum>
   </property>
  </action>
  <action name="actionAbrir">
   <property name="icon">
    <iconset resource="../resources/img.qrc">
     <normaloff>:/images/icons/open.svg</normaloff>:/images/icons/open.svg</iconset>
   </property>
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionCerrar">
   <property name="icon">
    <iconset resource="../resources/img.qrc">
     <normaloff>:/images/icons/close.svg</normaloff>:/images/icons/close.svg</iconset>
   </property>
   <property name="text">
    <string>Cerrar</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="icon">
    <iconset resource="../resources/img.qrc">
     <normaloff>:/images/icons/exit.svg</normaloff>:/images/icons/exit.svg</iconset>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="actionConfiguracion">
   <property name="icon">
    <iconset resource="../resources/img.qrc">
     <normaloff>:/images/icons/settings.svg</normaloff>:/images/icons/settings.svg</iconset>
   </property>
   <property name="text">
    <string>Configuraci√≥n</string>
   </property>
  </action>
  <action name="actionExportar">
   <property name="icon">
    <iconset resource="../resources/img.qrc">
     <normaloff>:/images/icons/export.svg</normaloff>:/images/icons/export.svg</iconset>
   </property>
   <property name="text">
    <string>Exportar</string>
   </property>
  </action>
  <action name="actionImprimir">
   <property name="icon">
    <iconset resource="../resources/img.qrc">
     <normaloff>:/images/icons/print.svg</normaloff>:/images/icons/print.svg</iconset>
   </property>
   <property name="text">
    <string>Imprimir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionRecientes">
   <property name="icon">
    <iconset resource="../resources/img.qrc">
     <normaloff>:/images/icons/recent.svg</normaloff>:/images/icons/recent.svg</iconset>
   </property>
   <property name="text">
    <string>Recientes</string>
   </property>
  </action>
  <action name="actionCopiar">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::EditCopy" resource="../resources/img.qrc">
     <normaloff>:/images/icons/copy.svg</normaloff>:/images/icons/copy.svg</iconset>
   </property>
   <property name="text">
    <string>Copiar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPegar">
   <property name="icon">
    <iconset resource="../resources/img.qrc">
     <normaloff>:/images/icons/paste.svg</normaloff>:/images/icons/paste.svg</iconset>
   </property>
   <property name="text">
    <string>Pegar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionCortar">
   <property name="icon">
    <iconset resource="../resources/img.qrc">
     <normaloff>:/images/icons/cut.svg</normaloff>:/images/icons/cut.svg</iconset>
   </property>
   <property name="text">
    <string>Cortar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionTutorial">
   <property name="icon">
    <iconset resource="../resources/img.qrc">
     <normaloff>:/images/icons/help.svg</normaloff>:/images/icons/help.svg</iconset>
   </property>
   <property name="text">
    <string>Tutorial</string>
   </property>
  </action>
  <action name="actionSobre_Nosotros_2">
   <property name="icon">
    <iconset resource="../resources/img.qrc">
     <normaloff>:/images/icons/info.svg</normaloff>:/images/icons/info.svg</iconset>
   </property>
   <property name="text">
    <string>Sobre Nosotros</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources/img.qrc"/>
 </resources>
 <connections/>
</ui>
